packages/bridge-controller/CHANGELOG.md

```diff
@@ -7,9 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [22.0.0]
+
 ### Changed
 
 - Bump `@metamask/controller-utils` to `^11.8.0` ([#5765](https://github.com/MetaMask/core/pull/5765))
+- **BREAKING:** Bump `@metamask/assets-controller` peer dependency to `^62.0.0` ([#5778](https://github.com/MetaMask/core/pull/5778))
```

Comment:
nit: not sure if order matters, but I usually put new entries on top.

Reply:
Fixed